docsrc/source/conf.py

```diff
@@ -114,9 +114,9 @@
 
 
 intersphinx_mapping = {
-    "numpy": ("http://docs.scipy.org/doc/numpy/", None),
-    "python": ("https://docs.python.org/", None),
-    "torch": ("https://pytorch.org/docs/main/", None),
+    "numpy": ("https://numpy.org/doc/1.26/", None),
```

Comment:
These versions need to be updated when we update versions.
Let's add this to our release TODO